run-stdio.sh

```
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Payload CMS MCP Server - Stdio Launcher
+# This script runs the MCP server in stdio mode for IDE integrations
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)